Mathlib/Analysis/Normed/Group/Tannery.lean

```diff
@@ -65,8 +65,8 @@ lemma tendsto_tsum_of_dominated_convergence {α β G : Type*} {𝓕 : Filter α}
     classical exact Eventually.exists <| hS _ (by positivity)
   have h1 : ∑' (k : (Tᶜ : Set β)), bound k < ε / 3 := by
     calc _ ≤ ‖∑' (k : (Tᶜ : Set β)), bound k‖ := Real.le_norm_self _
-         _ = ‖S - ∑ b ∈ T, bound b‖          := congrArg _ ?_
-         _ < ε / 3                            := by rwa [dist_eq_norm, norm_sub_rev] at hT
+         _ = ‖S - ∑ b ∈ T, bound b‖ := congrArg _ ?_
+         _ < ε / 3 := by rwa [dist_eq_norm, norm_sub_rev] at hT
```

Comment:
@harahu, this is a stylistic preference (i.e., putting the `:=` so that they line up vertically). Can you please revert this?